serving/src/main/java/feast/serving/service/ServingService.java

```diff
@@ -26,12 +26,75 @@
 import feast.serving.ServingAPIProto.GetOnlineFeaturesResponse;
 
 public interface ServingService {
+  /**
+   * Get information about the Feast serving deployment.
+   *
+   * <p>For Bigquery deployments, this includes the default job staging location to load
+   * intermediate files to. Otherwise, this method only returns the current Feast Serving backing
+   * store type.
+   *
+   * @param getFeastServingInfoRequest {@link GetFeastServingInfoRequest}
+   * @return {@link GetFeastServingInfoResponse}
+   */
   GetFeastServingInfoResponse getFeastServingInfo(
       GetFeastServingInfoRequest getFeastServingInfoRequest);
 
+  /**
+   * Get features from an online serving store, given a list of {@link
+   * feast.serving.ServingAPIProto.FeatureReference}s to retrieve, and list of {@link
+   * feast.serving.ServingAPIProto.GetOnlineFeaturesRequest.EntityRow}s to join the retrieved values
+   * to.
+   *
+   * <p>Features can be queried across feature sets, but each {@link
+   * feast.serving.ServingAPIProto.GetOnlineFeaturesRequest.EntityRow} must contain all entities for
+   * all feature sets included in the request.
+   *
+   * <p>This request is fulfilled synchronously.
+   *
+   * @param getFeaturesRequest {@link GetOnlineFeaturesRequest} containing list of {@link
+   *     feast.serving.ServingAPIProto.FeatureReference}s to retrieve and list of {@link
+   *     feast.serving.ServingAPIProto.GetOnlineFeaturesRequest.EntityRow}s to join the retrieved
+   *     values to.
+   * @return {@link GetOnlineFeaturesResponse} with list of {@link
+   *     feast.serving.ServingAPIProto.GetOnlineFeaturesResponse.FieldValues} for each {@link
+   *     feast.serving.ServingAPIProto.GetOnlineFeaturesRequest.EntityRow} supplied.
+   */
   GetOnlineFeaturesResponse getOnlineFeatures(GetOnlineFeaturesRequest getFeaturesRequest);
 
+  /**
+   * Get features from a batch serving store, given a list of {@link
```

Comment:
We are leaving the batch/historical renaming out of scope for the storage refactor right?